src/main/resources/liquibase/2.7.x.x/critical_range.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+	xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+	  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+	<property name="now" value="now()" dbms="postgresql" />
+
+	<changeSet author="cliff" id="1">
+		<validCheckSum>8:d30ab032096adb6b489edac763c55588</validCheckSum>
```

Comment:
you also don't need to add the validCheckSum , unless your modifying an existing changeset that already ran in production DBs